src/commands/switch.js

```diff
@@ -19,6 +24,21 @@ class SwitchCommand extends Command {
       }
     })
 
+    if(Object.keys(flags).length) {
```

Comment:
This logic should be activated only if the `userId` or `email` flags are provided, unrecognized flags should be ignored, allowing prompts to continue.